### PR TITLE
Ran rubocop --auto-correct

### PR DESCRIPTION
**Pull request by dlbucci**

### CODE REVIEW

File: spec/factories/assessments.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
     handin_filename "placeholder"
     handin_directory "placeholder"
     category_name "test category"
-    max_size 1024000
+    max_size 1_024_000
```
Comment (ymzong) [inline spec/factories/assessments.rb]: Wow I don't know Ruby allows this :stuck_out_tongue:

**Comment by icanberk** [comment]
Can we somehow add rubocoop checks to Travis? @ymzong

**Comment by ymzong** [comment]
@icanberk Sure, I can do that!

**Comment by jez** [comment]
All I can say is thank goodness for CI... otherwise PRs like this would be crazy scary :fearful:

**Pull request merged**